molo/commenting/admin_views.py

```diff
@@ -1,17 +1,25 @@
 from django.contrib import messages
-from django.http import HttpResponse
 from django.shortcuts import redirect
 from django.views.generic import FormView
 from django_comments.views.comments import post_comment
-from molo.commenting.admin_import_export import MoloCommentsResource
 from molo.commenting.forms import AdminMoloCommentReplyForm
-from molo.commenting.models import MoloComment
+from tasks import send_export_email
 from wagtail.contrib.modeladmin.views import IndexView
 from django.utils.translation import ugettext as _
 
 
 class MoloCommentsAdminView(IndexView):
+    def send_export_email_to_celery(self, email, arguments):
+        send_export_email.delay(email, arguments)
+
     def post(self, request, *args, **kwargs):
+        if not request.user.email:
+            messages.error(
+                request, _(
```

Comment:
@codiebeulaine we don't need the `_` here as this text is going to be shown to the admin user and we usually don't translate them